browsers/chrome-mv3/manifest.json

```diff
@@ -81,7 +84,9 @@
                 "/web_accessible_resources/*",
                 "/img/logo-small.svg",
                 "/public/css/autofill.css",
-                "/public/font/*"
+                "/public/font/*",
+                "/public/js/newtab.js",
```

Comment:
Does this have to be web-accessible? As it's just loaded by tracker-stats.html, maybe it's enough to have the html web-accessible?

Reply:
ahh nice! yeah that's a good change